buildSrc/src/main/kotlin/plugin/AndroidLibraryPlugin.kt

```diff
@@ -25,7 +25,7 @@ class AndroidLibraryPlugin : BasePlugin() {
         get() = {
             implementation(
                 Library.daggerHiltAndroid,
-                Library.coroutines
+                Library.coroutines,
```

Comment:
Probably need to get rid of that extra comma

Reply:
Okay, I'll do that